### PR TITLE
Release 2.11.0

### PR DESCRIPTION
**Pull request by d4straub**
Release 2.11.0, mostly updates for reference taxonomies and bug fixes. For details, see the change log.
(The release is planned for 6th of August, because I'll be in holiday next week ;) )

Still to do:
- [x] Test run on AWS

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Check for unexpected warnings in debug mode (`nextflow run . -profile debug,test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,29 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## nf-core/ampliseq version 2.11.0 - 2024-08-06
+
+### `Added`
+
+- [#765](https://github.com/nf-core/ampliseq/pull/765) - Added version R09-RS220 of curated GTDB 16S taxonomy: `sbdi-gtdb=R09-RS220-1` or `sbdi-gtdb` as parameter to `--dada_ref_taxonomy`
+- [#766](https://github.com/nf-core/ampliseq/pull/766) - Added version 10 of Unite as options for `--sintax_ref_taxonomy`
```
Comment (erikrikarddaniel) [inline CHANGELOG.md:11]:
```suggestion
- [#766](https://github.com/nf-core/ampliseq/pull/766) - Added version 10 of Unite as parameter for `--sintax_ref_taxonomy`: `unite-fungi=10.0` and `unite-alleuk=10.0`
```

File: conf/test_its_dada_taxonomy.config [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    Use as follows:
+        nextflow run nf-core/ampliseq -profile test_sintax,<docker/singularity> --outdir <OUTDIR>
+
+----------------------------------------------------------------------------------------
+*/
+
+params {
+    config_profile_name = 'Test sintax profile'
+    config_profile_description = 'Minimal test dataset to check pipeline function when using sintax for taxonomy assignment'
```
Comment (erikrikarddaniel) [inline conf/test_its_dada_taxonomy.config:15]:
```suggestion
    config_profile_description = 'Minimal test dataset to check pipeline function for ITS data with the DADA2 taxonomy classifier'
```

File: docs/output.md [hunk 3]
```diff
@@ -200,7 +202,7 @@ Optionally, ASV sequences can be filtered for rRNA sequences identified by Barrn
 
 #### Length filter
 
-Optionally, a length filter can be used to reduce potential contamination after ASV computation. For example with 515f and 806r primers the majority of 16S rRNA amplicon sequences should have a length of 253 bp and amplicons vary significantely are likely spurious.
+Optionally, a length filter can be used to reduce potential contamination after ASV computation. For example with 515f and 806r primers the majority of 16S rRNA amplicon sequences should have a length of 253 bp and amplicons that vary significantly are likely spurious.
```
Comment (erikrikarddaniel) [inline docs/output.md:205]:
```suggestion
Optionally, a length filter can be used to reduce potential contamination after ASV computation. For example with 515f and 806r primers the majority of 16S rRNA amplicon sequences should have a length of 253 bp and amplicons that differ significantly from this are likely spurious.
```

**Review by erikrikarddaniel — APPROVED** [review]
A few small things, mostly correcting myself.

**Comment by d4straub** [comment]
Thanks, I added your suggestions!

**Review by mashehu — APPROVED** [review]

**Comment by d4straub** [comment]
Thanks for the reviews! I'll wait for the test run on AWS to complete before merging.

**Pull request merged**